TextFieldEffects/TextFieldsDemo/ExampleTableViewController.swift

```diff
@@ -19,26 +19,84 @@ class ExampleTableViewController : UITableViewController, UITextFieldDelegate {
    textFields prepopulated with the name "Raul" (for testing purposes)
    */
   let prefillTextFields = false
+  var cells = [Any](repeating: 0, count: 2)
 
   override func viewDidLoad() {
     super.viewDidLoad()
-    guard prefillTextFields == true else { return }
+//    guard prefillTextFields == true else { return }
```

Comment:
I know its example but I don't think you need to keep commented code.

Reply:
you are absolutely right, I forgot about this